.changeset/famous-kids-smell.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
what is this file?

Reply:
That seems odd? What is the CI check doing?

Reply:
See bullet no. 2 [here](https://github.com/Shopify/shopify-app-js/blob/next/RELEASING.md#releasing-shopify-app-js-packages)

This file is the result of running `yarn changeset --empty`, which is indicating to `changesets` that the dev doesn't want to anything to the `CHANGELOG.md` files for this PR.

The CI check is seeing if there's a changeset file being added for the PR - it's a programmatic check to see if the dev has added an entry for the CHANGELOG, like the CI check for the tick box in the PR template!